plugins/mysql/mysqlembed.cpp

```diff
@@ -427,7 +427,7 @@ class MySQLConnection : public CInterface
         ForEachItemInRev(idx, globalCachedConnections)
         {
             MySQLConnection &cached = globalCachedConnections.item(idx);
-            if (now - cached.created > maxAge)
+            if (!maxAge || (now - cached.created > maxAge))
```

Comment:
|| is right I think

The test is not quite redundant in that now==cached.created is possible.